app/models/movie.rb

```diff
@@ -21,4 +22,8 @@ def self.save_movie_from_imsdb_node(node)
       puts "#{movie.title} could not be saved: #{movie.errors.full_messages}"
     end
   end
+
+  def self.script_scanner
```

Comment:
Is this being used here? if not, please remove it and the require statement above.